frontend/apps/app/components/ProjectLayout/ProjectHeader/ProjectHeader.tsx

```diff
@@ -66,9 +66,11 @@ export const ProjectHeader: FC<ProjectHeaderProps> = async ({
       <TabsList className={styles.tabsList}>
         {PROJECT_TABS.map((tab) => {
           const Icon = tab.icon
+          const isSchemaTab = tab.value === 'schema'
+          const isDisabled = isSchemaTab && !project.schemaPath
           let href: string
 
-          // For all tabs, use the ref/[branchOrCommit] routes
+          // For all tabs, use the ref/[branchOrCommit] routes (important-comment)
```

Comment:
I don't think this change makes sense. Since the comment also explains the "What," I'll go ahead and delete it.


```suggestion
```